api/src/main/java/org/openmrs/Allergy.java

```diff
@@ -305,27 +305,27 @@ public AllergyReaction getAllergyReaction(Integer allergyReactionId) {
 				return reaction;
 			}
 		}
-		
+
```

Comment:
Most of your changes are just whitespaces, could you resolve that, here and in many other places?